test/foundry/integration/ITransparentUpgradeableProxy.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+interface ITransparentUpgradeableProxy {
```

Comment:
Cant we import these interfaces instead of adding them into the repo?

Reply:
This one is not available in OZ package